### PR TITLE
mention additional resources in README

### PR DESCRIPTION
**Pull request by jgravois**
addresses #71 

i like the idea of putting the information directly in the README and just linking to Esri's generic boilerplate CONTRIBUTING doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,13 @@ Refer to the README files in each subdirectory of this repo for specific instruc
...
 
 ## Contributing
-Anyone and everyone is welcome to contribute. We accept pull requests. Feel free to:  
-* Report issues
-* Contribute code
-* Improve documentation
+
+Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
+
+We use issues and pull requests to address problems found in the [ArcGIS API for Javascript](http://js.arcgis.com/) here.  If you're asking for help please utilize the supplementary resources below.
```
Comment (dasa) [inline README.md]: We don't actually use this repo to address problems in the API though.

**Comment by dasa** [comment]
I think having a CONTRIBUTING.md is a good idea too so that it's seen when opening a Issue or PR.

File: README.md (hunk 1 above)
Comment (jgravois) [inline README.md]: what about #33? 

either way, i think it'd be fine to just ditch that sentence.

> ~~Issues and pull requests to address problems found in the [ArcGIS API for Javascript](http://js.arcgis.com/) here.~~ If you're asking for help please utilize the supplementary resources below.

i hadn't thought of the fact that users are directed to CONTRIBUTING.md doc when they open issues and pull requests. thats a good point.

Comment (dasa) [inline README.md]: 33?



> asking for help

Maybe distinguish between help with the API vs. the files in this repo?

File: README.md [hunk 2]
```diff
@@ -17,19 +17,17 @@ A collection of resources for developers using the [ArcGIS API for JavaScript](h
...
 * [JSHint](http://www.jshint.com/)
 * [TypeScript](http://www.typescriptlang.org/)
 
 ## Contributing
-Anyone and everyone is welcome to contribute. We accept pull requests. Feel free to:  
-* Report issues
-* Contribute code
-* Improve documentation
+
+Please see our [guidelines for contributing](contributing).
```
Comment (dasa) [inline README.md]: 404

File: README.md (hunk 1 above)
Comment (jgravois) [inline README.md]: https://github.com/Esri/jsapi-resources/issues/33 is an example of a situation in which someone proposed a fix for a problem in the API in this repo.

in general, you're right. distinguishing exactly what is being tracked here and what types of contributions are the most appropriate and guiding users to alternative (and superior resources) for other help is what's needed.

**Pull request merged**